tests/test_convergence.py

```diff
@@ -0,0 +1,101 @@
+import numpy as np
+from ase import Atoms
+import pytest
+from numpy.testing import assert_allclose
+from anisoap.representations import EllipsoidalDensityProjection
+import metatensor
+
+from scipy.special import sph_harm
+
+
+class TestGaussianConvergence:
+    sigmas = np.array([1.0, 2.0, 3.0])
+
+    @pytest.mark.parametrize("sigma", sigmas)
+    def test_single_atom_isotropic_convergence(self, sigma):
+        """
+        Test that coefficients are correct, such that the approximation using these coefficients in the expansion
+        can reasonably recreate the original atomic gaussian (in this case, an unnormalized gaussian with sigma=1)
+        """
+
+        frame = Atoms(positions=np.array([[0, 0, 0]]), numbers=[0])
+        frame.arrays["quaternions"] = np.array([[1, 0, 0, 0]])
+
+        frame.arrays[r"c_diameter[1]"] = 2 * sigma * np.ones(1)
+        frame.arrays[r"c_diameter[2]"] = 2 * sigma * np.ones(1)
+        frame.arrays[r"c_diameter[3]"] = 2 * sigma * np.ones(1)
+        frames = [frame]
+
+        # An rgw that's a little bigger than the atom gaussian sigma converges well and is sufficient for this test.
+        rgw = sigma + 0.5
+
+        max_radials = range(10)
+        errs = []
+        for max_radial in max_radials:
+            HYPER_PARAMETERS = {
```

Comment:
What are hyper parameters

Reply:
Hyperparameters are parameters that you choose to run your model with.
This is opposed to parameters, which is what your model tries to find to best fit your data.

e.g. if you're trying to fit a polynomial to a bunch of points, like
$y=a_0x^0 + a_1x^1 + ... + a_nx^n$, 

n is a hyperparameter that you choose beforehand, to determine what order polynomial you want to fit to your data (n=1 would be a linear fit, n=2 would be a quadratic fit, etc.). After you choose your hyperparameter n, the model determines $a_0, a_1, ...$, which are the parameters that best fit the data.

Reply:
So we can choose the number of hyperparameters to determine the coefficient of bases? Would deciding what parameters affect the computational cost and accuracy of prediction?

Reply:
Absolutely. Choosing higher values for hyperparameters gives us higher-resolution descriptors (i.e more coefficients) but is more computationally expensive.